common/djangoapps/entitlements/rest_api/v1/views.py

```diff
@@ -315,7 +315,10 @@ class EntitlementEnrollmentViewSet(viewsets.GenericViewSet):
         - Unenroll
         - Switch Enrollment
     """
-    authentication_classes = (JwtAuthentication, SessionAuthentication,)
+    authentication_classes = (
+        JwtAuthentication,
+        SessionAuthenticationAllowInactiveUser,
```

Comment:
Are there any security concerns?

Reply:
@julianajlk @pshiu: [inform] I created a DEPR ticket around this issue: https://github.com/openedx/public-engineering/issues/165

Reply:
Thanks, @robrap! On your questions, @julianajlk:
* Yes, users who have not verified their email are considered inactive even when logged in! I was confused about this as well. They are marked `is_active = False` in the Django user model in the database.
* I didn't have any security concerns because `JwtAuthentication` already allows inactive users to use this endpoint. But I will raise this with the Security Working Group this afternoon.